domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/domene/iay/inntektsmelding/Inntektsmelding.java

```diff
@@ -442,6 +442,8 @@ public String toString() {
             ", refusjonOpphører=" + refusjonOpphører +
             ", innsendingsårsak= " + innsendingsårsak +
             ", innsendingstidspunkt= " + innsendingstidspunkt +
+            ", graderinger= " + graderinger +
+            ", naturalytelse= " + naturalYtelser +
             '>';
```

Comment:
Er det ok å logge dette her?

Reply:
Jeg antar det er ok, men vi logger visst mye ting i abakus som ikke er ok så tror det er greit å forhøre seg med PE eller funk. her. Men er greit for meg 👍